index.d.ts

```diff
@@ -116,8 +128,8 @@ export interface DailyParticipant {
   owner: boolean;
 
   // video element info (iframe-based calls using standard UI only)
-  cam_info: {} | DailyVideoElementInfo;
-  screen_info: {} | DailyVideoElementInfo;
+  cam_info: DailyVideoElementInfo;
```

Comment:
not sure what the empty type with the union was previously. This might need to be `Partial<DailyVideoElementInfo>` if the intent was to signal that the keys could be optional